HoneyBadgers.WebApp/Controllers/AccountController.cs

```diff
@@ -47,14 +53,22 @@ public async Task<IActionResult> Register(RegisterViewModel model)
                     Email = model.Email
 
                 };
+                LogContext.PushProperty("UserName", model.Email);
```

Comment:
co to robi, bo nie kojarzę?

Reply:
Pushuje property w tym wypadku nazwę użytkownika do customowej kolumny UserName w tabeli :)